nep framework/tests/SampleTest.php

```diff
@@ -0,0 +1,10 @@
+<?php
+namespace nepframework;
```

Comment:
Proper template

```
<?php
namespace Nepframework;

class SampleTest extends \PHPUnit_Framework_TestCase
{
    public function testAssertTrue()
    {
        $this->assertTrue(TRUE);
    }
}
```

Reply:
This is great, clean and more clear IMO.

Reply:
Yes i did implement it.

Reply:
Thank you for the guidance.

Reply:
you are welcome. If you check other issues that I have created there are links for using different tools to add improvements in your project via Travis or Code climate, etc.